packages/relay/src/lib/eth.ts

```diff
@@ -1750,13 +1750,21 @@ export class EthImpl implements Eth {
       EthImpl.ethGetTransactionReceipt,
       requestIdPrefix,
     );
+
     if (cachedLog) {
+      const block = await this.getBlockByHash(cachedLog.blockHash, false);
+      const gasPriceForTimestamp = await this.getFeeWeibars(
```

Comment:
Please add an acceptance test case under `rpc_batch1.spec.ts` that submits an HTS transfer transaction and verifies the transactionReceipt has the expected value